.github/workflows/gradlebuild.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           settings-path: ${{ github.workspace }} # location for the settings.xml file
 
       - name: Build with Gradle
-        run: gradle build
+        run: gradle shadowRemapJar
```

Comment:
reasoning behind this?

Reply:
gradlew build caused some unexplainable license errors even though the license header is the same (and was automatically generated by the same program on top of that!) Also, the README states to use this.